StudyHub/StudyHub/Program.cs

```diff
@@ -2,9 +2,15 @@
 using StudyHub.DAL.EF;
 using StudyHub.DAL.Repositories.Interfaces;
 using StudyHub.DAL.Repositories;
+using StudyHub.BLL.Profiles;
+using StudyHub.BLL.Interfaces;
+using StudyHub.BLL.Services;
 
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddAutoMapper(typeof(AssignmentTaskProfile).Assembly);
+builder.Services.AddAutoMapper(typeof(OptionsProfile).Assembly);
```

Comment:
I don't think that you need to add mapper for every profile
Check documentation about this